src/path.js

```diff
@@ -16,6 +16,290 @@ function Path() {
     this.strokeWidth = 1;
 }
 
+function roundDecimal(float, places) {
+    return +(Math.round(float + 'e+' + places) + 'e-' + places);
```

Comment:
The type conversions here may slow this down. Might I suggest
```js
const shift = Math.pow(10,places);
return (Math.round(float*shift)/shift);
```
instead.

Reply:
That seems to have introduced a rounding issue

Reply:
I'll take a closer look at this and run the unicode test suite to see what impact it has. Maybe it could even make things better in some cases. We'll see.

Reply:
From what I see, the `roundDecimal` is used at 2 places, for different reasons: 
- at path parsing, where all precision shall be kept (?)
- at serialisation, where it shall treated number as (truncated) String, not a rounded Number (which is sometime impossible due to floating point precision)

Reply:
So when I adapt the test case, we don't fail any other tests and the number of passed tests in the Unicode test suite doesn't change, but I really think `M0.575` should change to `M0.58` when changing the `decimalPlaces` option from 3 to 3, and not `M0.57`. Why would we ever want to cut off decimal places instead of actual rounding?
I think that's why I chose to do it the way I did in the first place, because I always ended up with some weird rounding errors. But I'm open to a solution that feels less hacky and still produces correct values.

Reply:
for now, I reversed the change and rebased on master

Reply:
That should be functionally equivalent to the rounding function you had originally as they do the exact same thing.

Reply:
I don't know why the test is failing with a wrong value then, either 🤔

Reply:
I was able to get around that rounding issue by adding `Number.EPSILON` before multiplying
```js
const shift = Math.pow(10, places);
return (Math.round((float + Number.EPSILON) * shift) / shift);
```
which makes all tests pass again - but when I tried with different values, `10.075` rounds to `10.07` instead of `10.08`.
My original implementation was the only one so far that seems to work for all cases. I'd say we leave it like that before introducing precision issues.